api/auth/auth.controller.js

```diff
@@ -62,7 +65,10 @@ class AuthController {
     })
   }
 
-  signup(email, password) {
+  signup(request) {
+    const { payload } = request || {}
```

Comment:
The same:
destructuring and default value setting can be made at the function's parameter place.